src/components/completing-party/CompletingParty.stories.ts

```diff
@@ -31,5 +32,8 @@ Default['args'] = {
       streetAddressAdditional: 'Test Street Additional'
     }
   },
-  editing: true
+  enableAddEdit: false,
+  addressSchema: DefaultSchema,
```

Comment:
Looks good now:

https://github.com/bcgov/bcrs-shared-components/assets/122301442/e9c639f2-dbf3-4cd2-bc2d-f6240ba7b070

Reply:
Well done Karim! You're a hero ❤️